blockchain_connector/Dockerfile

```diff
@@ -120,24 +116,112 @@ RUN mkdir -p build \
   && cmake .. -DUNTRUSTED_ONLY=1 \
   && make
 
-#install avalon common module
+
+# -------------=== common/python build ===-------------
+
+#Build common/python intermediate docker image
+FROM python_image as common_python_image
+
+COPY VERSION /project/avalon/
+COPY ./bin /project/avalon/bin
+
+ENV TCF_HOME=/project/avalon
+
+COPY ./common/python /project/avalon/common/python
+
 WORKDIR /project/avalon/common/python
 
 RUN echo "Building Avalon Common Python\n" \
-  && make
+ && make
+
+
+# -------------=== common/crypto_utils build ===-------------
+
+#Build common/cpp intermediate docker image
+FROM python_image as common_crypto_utils_image
+
+RUN apt-get update \
```

Comment:
Can we use the updated packages from python_image instead of running `apt-get update` here. In general I've noticed that `apt-get update` consumes time during build.